pyonfx/utils.py

```diff
@@ -44,12 +44,7 @@ def all_non_empty(
         Returns:
             A list containing lines_chars_syls_or_words without objects with duration equals to zero or blank text (no text or only spaces).
         """
-        out = []
-        for obj in lines_words_syls_or_chars:
-            if obj.text.strip() and obj.duration > 0:
-                out.append(obj)
-        return out
-        # return [obj for obj in lines_words_syls_or_chars if obj.text.strip() and obj.duration > 0]
+        return [obj for obj in lines_words_syls_or_chars if obj.text.strip() and obj.duration > 0]
```

Comment:
Linters do not complain about list comprehension?

Reply:
They did but the commit https://github.com/CoffeeStraw/PyonFX/pull/33/commits/1bceb5bd6105edf5417fdca6b21c02482f36c8b2 fix it